From 8f195a7a2af6511cd7b732a7641bca3771d5b431 Mon Sep 17 00:00:00 2001
From: Jeremie Graulle <jeremiex.graulle@intel.com>
Date: Tue, 2 Apr 2013 13:05:52 +0200
Subject: [DO NOT UPSTREAM][PORT FROM MAIN] libmedia_helper: Host static lib compil for UT

BZ: 93434

Need to compile this project in host static lib mode
in order to compile system and unit test of the
"Configurable HAL" component.

Add a host static lib compilation in libmedia Android.mk.

Change-Id: Ibf30b297abc4d9512bf4d9154c411b18c7147371
Orig-Change-Id: I96eca243cb914f45aad54b9cded556845dbab1f4
Signed-off-by: Jeremie Graulle <jeremiex.graulle@intel.com>
---
 media/libmedia/Android.mk |   22 ++++++++++++++++++++++
 1 files changed, 22 insertions(+), 0 deletions(-)

diff --git a/media/libmedia/Android.mk b/media/libmedia/Android.mk
index 2c0c3a5..f27e9ce 100644
--- a/media/libmedia/Android.mk
+++ b/media/libmedia/Android.mk
@@ -1,5 +1,8 @@
 LOCAL_PATH:= $(call my-dir)
 
+# Build for helper target
+#########################
+
 include $(CLEAR_VARS)
 
 LOCAL_SRC_FILES:= \
@@ -9,6 +12,25 @@ LOCAL_MODULE_TAGS := optional
 
 include $(BUILD_STATIC_LIBRARY)
 
+# Build for helper host test
+############################
+
+include $(CLEAR_VARS)
+
+ifeq ($(strip $(INTEL_MUSIC_OFFLOAD_FEATURE)),true)
+  LOCAL_CFLAGS += -DINTEL_MUSIC_OFFLOAD_FEATURE
+endif
+
+LOCAL_SRC_FILES:= \
+    AudioParameter.cpp
+LOCAL_MODULE:= libmedia_helper_host
+LOCAL_MODULE_TAGS := optional
+
+include $(BUILD_HOST_STATIC_LIBRARY)
+
+# Build for main lib target
+###########################
+
 include $(CLEAR_VARS)
 
 LOCAL_SRC_FILES:= \
-- 
1.7.4.1

